file.c

```diff
@@ -5321,60 +5321,83 @@ test_check(int n, int argc, VALUE *argv)
- *    "k"  | boolean | True if file1 exists and has the sticky bit set
- *    "l"  | boolean | True if file1 exists and is a symbolic link
- *    "M"  | Time    | Last modification time for file1
- *    "o"  | boolean | True if file1 exists and is owned by
- *         |         | the caller's effective uid
- *    "O"  | boolean | True if file1 exists and is owned by
- *         |         | the caller's real uid
- *    "p"  | boolean | True if file1 exists and is a fifo
- *    "r"  | boolean | True if file1 is readable by the effective
- *         |         | uid/gid of the caller
- *    "R"  | boolean | True if file is readable by the real
- *         |         | uid/gid of the caller
- *    "s"  | int/nil | If file1 has nonzero size, return the size,
- *         |         | otherwise return nil
- *    "S"  | boolean | True if file1 exists and is a socket
- *    "u"  | boolean | True if file1 has the setuid bit set
- *    "w"  | boolean | True if file1 exists and is writable by
- *         |         | the effective uid/gid
- *    "W"  | boolean | True if file1 exists and is writable by
- *         |         | the real uid/gid
- *    "x"  | boolean | True if file1 exists and is executable by
- *         |         | the effective uid/gid
- *    "X"  | boolean | True if file1 exists and is executable by
- *         |         | the real uid/gid
- *    "z"  | boolean | True if file1 exists and has a zero length
- *
- *  Tests that take two files:
- *
- *    "-"  | boolean | True if file1 and file2 are identical
- *    "="  | boolean | True if the modification times of file1
- *         |         | and file2 are equal
- *    "<"  | boolean | True if the modification time of file1
- *         |         | is prior to that of file2
- *    ">"  | boolean | True if the modification time of file1
- *         |         | is after that of file2
+ *    test(char, path0, path1 = nil) -> object
+ *
+ *  Performs a test on one or both of the <i>filesystem entities</i> at the given paths
+ *  `path0` and `path1`:
+ *
+ *  - Each path `path0` or `path1` points to a file, directory, device, pipe, etc.
+ *  - Character `char` selects a specific test.
+ *
+ *  The tests:
+ *
+ *  - Each of these tests operates only on the entity at `path0`,
+ *    and returns `true` or `false`;
+ *    for a non-existent entity, returns `false` (does not raise exception):
+ *
+ *      | Character | Test |
+ *      |:---------:|:----- |
+ *      | <tt>'b'</tt> | Whether the entity is a block device. |
+ *      | <tt>'c'</tt> | Whether the entity is a character device. |
+ *      | <tt>'d'</tt> | Whether the entity is a directory. |
+ *      | <tt>'e'</tt> | Whether the entity is an existing entity. |
+ *      | <tt>'f'</tt> | Whether the entity is an existing regular file. |
+ *      | <tt>'g'</tt> | Whether the entity's setgid bit is set. |
+ *      | <tt>'G'</tt> | Like <tt>'g'</tt>, but also tests whether the ownership is equal to the caller's. |
```

Comment:
`G` is unrelated to `g`.
```suggestion
 *      | <tt>'G'</tt> | Whether the entity's group ownership is equal to the caller's. |
```

Reply:
Fixed.